### ISSUE TITLE
URI.decode_www_form broken in 9k

### ISSUE THREAD

**Issue by iconara** [jruby/jruby#2105]
```
irb> URI.decode_www_form("projection=x")
NoMethodError: undefined method `scrub!' for "projection":String
	from /Users/theo/.rvm/rubies/jruby-head/lib/ruby/2.1/uri/common.rb:1000:in `decode_www_form'
	from org/jruby/RubyArray.java:1546:in `each'
	from /Users/theo/.rvm/rubies/jruby-head/lib/ruby/2.1/uri/common.rb:998:in `decode_www_form'
	from (irb):3:in `evaluate'
	from org/jruby/RubyKernel.java:962:in `eval'
	from org/jruby/RubyKernel.java:1282:in `loop'
	from org/jruby/RubyKernel.java:1092:in `catch'
	from org/jruby/RubyKernel.java:1092:in `catch'
	from /Users/theo/.rvm/rubies/jruby-head/bin/irb:13:in `__script__'
```

```
$ ruby -v
jruby 9000.dev-SNAPSHOT (2.1.2p142) 2014-11-05 624823a Java HotSpot(TM) 64-Bit Server VM 25.0-b70 on 1.8.0-b132 +jit [darwin-x86_64]
$ java -version
java version "1.8.0"
Java(TM) SE Runtime Environment (build 1.8.0-b132)
Java HotSpot(TM) 64-Bit Server VM (build 25.0-b70, mixed mode)
```

**Comment by dougfarre** [comment]
+1

**Comment by enebo** [comment]
@iconara @dougfarre 

```ruby
require 'uri'

URI.decode_www_form("projection=x")
```

This works for me.   Can you guys verify?

**Comment by iconara** [comment]
Looks like it's fixed in `jruby-9.0.0.0.pre1`
